libraries/stdlib/samples/test/samples/system/timing/timing.kt

```diff
@@ -0,0 +1,26 @@
+package samples.system.timing
+
+import samples.*
+import kotlin.system.*
+import kotlin.test.*
+
+class Timings {
+
+    @Sample
+    fun measureBlockTimeMillis() {
+        val timeInMillis = measureTimeMillis {
+            (0..10).reduce(Int::plus)
```

Comment:
I suppose given the resolution of millisecond timer this result is going to be 0 every time. Perhaps we can find some other task that lasts for 10-20 ms in Kotlin playground?

Reply:
Cool. I'll try to find something else to use.